components/backup-stream/src/checkpoint_manager.rs

```diff
@@ -89,14 +96,12 @@ impl SubscriptionManager {
                 sub.flush().await
             };
 
-            match send_all.await {
-                Err(grpcio::Error::RemoteStopped) => {
+            if let Err(err) = send_all.await {
+                let can_retry = matches!(&err, grpcio::Error::RpcFailure(rpc_err) if rpc_err.code() == RpcStatusCode::UNAVAILABLE);
```

Comment:
Why this error can retry?

Reply:
By default `grpc-go` would retry it. (Should we also add `RESOURCE_EXHAUSTED`?)

https://pkg.go.dev/github.com/grpc-ecosystem/go-grpc-middleware/retry#pkg-variables

Reply:
And it seems there is transparent retry [inside gRPC implementation](https://github.com/grpc/proposal/blob/master/A6-client-retries.md)(Though I have neither checked nor configured that.), perhaps we can remove this? What do you think? @BusyJay

Reply:
I have no idea. I'm just curious why only this error can be retried.

Reply:
Well, I have removed the retry, because the client would retry sooner when the connection closed.